docs/README.md

```diff
@@ -0,0 +1,37 @@
+# River UI
+
+River UI is a graphical user interface for the [River job queue](https://github.com/riverqueue/river). It lets users view and manage jobs without having to resort to querying the database or the command line.
+
+## Installation
+
+A working River database is required for the UI to start up properly. See [running River migrations](https://riverqueue.com/docs/migrations), and make sure a `DATABASE_URL` is exported to env.
+
+```sh
+$ go install github.com/riverqueue/river/cmd/river@latest
+$ river migrate-up --database-url "$DATABASE_URL"
```

Comment:
Not for immediate discussion here, but I realized while trying these instructions that one nice thing about Direnv is that you can use the env vars with tools that are _not_ the main project, which is definitely nice.

Reply:
Yeah, I think we should make a quick attempt to see if we can use direnv with just a thin wrapper layer for the frontend (which requires `.env` files).